ibis/tests/expr/test_table.py

```diff
@@ -238,11 +238,37 @@ def test_projection_array_expr(table):
 
 
 def test_mutate(table):
-    one = table.f * 2
-    foo = (table.a + table.b).name('foo')
-
-    expr = table.mutate(foo, one=one, two=2)
-    expected = table[table, foo, one.name('one'), ibis.literal(2).name('two')]
+    expr = table.mutate(
+        [
```

Comment:
Can we remove the `[]`s? While that is supported, it's not the preferred way of passing a sequence of columns; you can pass them each as a positional argument.

Reply:
This is fine, no reason to wait to merge this :)

Reply:
> you can pass them each as a positional argument.

I don't think this is true? The signature of `mutate` is `mutate(exprs=None, **mutations)` - it's not variadic. Or were you asking if we could make it variadic (I think that'd be a good idea).

Reply:
You're right, I was thinking of select!